README.md

```diff
@@ -1,31 +1,35 @@
-Contao eRecht24 Rechtstexte
+Contao Rechtstexte für eRecht24
```

Comment:
_für_? Kommen die Rechtstexte nicht _von_ eRecht24? :)

Reply:
Ja, ich schrieb Christian eben schon via E-Mail, dass dieses `für` intern gewünscht wurde um klarer zu signalisieren, dass die Extension nicht von eRecht24 selber entwickelt wird. Wenn ihr eine bessere Idee habt, könnt ihr es gerne auch anders benennen.

Reply:
Vielleicht _eRecht24 Rechtstexte für Contao_? Oder signalisiert das zuwenig, dass es nicht von eRecht24 selbst programmiert wird.

Reply:
- eRecht24 Rechtstexte in Contao nutzen
- eRecht24 Rechtstexte für Contao nutzen
- eRecht24 API-Verbindung für Contao

Reply:
- Contao-Anbindung für Rechtstexte von eRecht24
- Contao-Schnittstelle zum Abruf von eRecht24 Rechtstexten